openapi-cli/src/test/resources/generators/client/swagger/pdf_payload.yaml

```diff
@@ -0,0 +1,23 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: OpenApi Petstore
+  license:
+    name: MIT
+servers:
+  - url: http://petstore.{host}.io/v1
+    description: The production API server
+paths:
+  /pets:
+    post:
+      summary: Create a pet
+      operationId: createPet
+      requestBody:
+        content:
+          application/pdf:
+            schema:
+              format: binary
+              type: string
+      responses:
+        '201':
+          description: Null response
```

Comment:
Add new line there :)

Reply:
Added in https://github.com/ballerina-platform/ballerina-openapi/pull/692/commits/1151944387857250e3b9f366f859863fac787fd6